### PR TITLE
Weekly merge 2

### PR DESCRIPTION
**Pull request by dimkouv**
## Solution

### CODE REVIEW

File: integration-tests/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/smartcontractkit/ccip/integration-tests
+module github.com/smartcontractkit/chainlink/integration-tests
```
Comment (AnieeG) [inline integration-tests/go.mod]: This should be github.com/smartcontractkit/ccip/integration-tests

File: integration-tests/go.mod [hunk 2]
```diff
@@ -491,8 +483,4 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
-
-	github.com/smartcontractkit/chainlink/integration-tests => ../integration-tests
-
-	k8s.io/apimachinery => k8s.io/apimachinery v0.25.11
```
Comment (AnieeG) [inline integration-tests/go.mod]: This is required

**Review by AnieeG — DISMISSED** [review]

**Comment by makramkd** [comment]
Looks good - can we squash the fix commits into 1?

**Review by makramkd — APPROVED** [review]

**Comment by RensR** [comment]
still quite a few to be squashed

**Review by RensR — APPROVED** [review]

**Pull request merged**